src/test/java/org/openstreetmap/atlas/tags/annotations/extraction/AltitudeExtractorTest.java

```diff
@@ -0,0 +1,43 @@
+package org.openstreetmap.atlas.tags.annotations.extraction;
+
+import java.util.Optional;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openstreetmap.atlas.geography.Altitude;
+
+/**
+ * Unit test for {@link AltitudeExtractor}.
+ *
+ * @author bbreithaupt
+ */
+public class AltitudeExtractorTest
```

Comment:
Nice and simple! Could you extend those to all the unit use cases? KM, MI and NMI? And also some upper case, lower case and mixed case?

Reply:
I have not added other units to the AltitudeExtractor tests, because the Altitude class only takes meters currently.

Reply:
Oh yes. Thanks!